compile-with-docker-all.sh

```diff
@@ -1,30 +1,61 @@
-#!/bin/sh
+#!/bin/bash
 #export DOCKER_DEFAULT_PLATFORM=linux/amd64
+
 IMAGE_NAME="uvk5"
-rm "${PWD}/compiled-firmware/*"
+DEST_DIR="compiled-firmware/"
+LOCL_DIR="${PWD}/${DEST_DIR}/"
+ALWAYSIN="ENABLE_NOAA=0 NABLE_UART=1"
```

Comment:
NABLE_UART typo

Reply:
Thanks, a mistake that can cost a bricked device! Isn't it, right?

Reply:
Thankfully these devices are pretty hard to brick. UART is not 100% needed for normal operation unless you want to use Chirp, etc.